index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
```

Comment:
The `<head>` information in this project are still the Vite default (take a look at your tab to see the effects of this). Please update the title, meta description and favicon to match your actual project.